lib/services/android-project-service.ts

```diff
@@ -517,12 +517,18 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		if (this.$androidToolsInfo.getToolsInfo().androidHomeEnvVar) {
 			const gradlew = this.$hostInfo.isWindows ? "gradlew.bat" : "./gradlew";
 
+			const localArgs = [...gradleArgs];
+			if (this.$logger.getLevel() === "INFO") {
```

Comment:
let's keep it this way, `$loggingLevels` has a different purpose. I'll handle the logging levels in a separate PR and will change this code with a constant in it.